### PR TITLE
feat: abpt v2

### PR DESCRIPTION
**Pull request by grothem**
## General Changes

- Preparing for ABPT v2 token. See governance post for more info: https://governance.aave.com/t/bgd-abpt-balancer-v2-migration-plan/8381

TODO:
- [x] Merge utils and create package https://github.com/aave/aave-utilities/pull/573
- [ ] Once proposal is live, update the warning message in `staking.staking.tsx` for the ABPT panel and add a `Learn more` to link to proposal.

### CODE REVIEW

**Comment by grothem** [comment]
Link T-12905

**Review by MareskoY — APPROVED** [review]

File: pages/staking.staking.tsx [hunk 1]
```diff
@@ -208,20 +238,75 @@ export default function Staking() {
...
+                item
+                xs={12}
+                lg={6}
+                sx={{ display: { xs: !isStkBpt ? 'none' : 'block', lg: 'block' } }}
+              >
+                {showAbptPanel && (
+                  <StakingPanel
+                    stakeTitle="ABPT"
+                    stakedToken="ABPT"
+                    maxSlash={stkBpt?.maxSlashablePercentageFormatted || '0'}
```
Comment (foodaka) [inline pages/staking.staking.tsx:265]: good reading now from contracts

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "test:coverage": "jest --coverage"
   },
   "dependencies": {
-    "@aave/contract-helpers": "1.23.1",
+    "@aave/contract-helpers": "1.23.2-6b938c51f8513000aae669d310ab5015e957abd7.0",
```
Comment (foodaka) [inline package.json:33]:
```suggestion
    "@aave/contract-helpers": "1.24.0",
```

File: package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
...
     "@aave/math-utils": "1.23.1",
```
Comment (foodaka) [inline package.json:34]:
```suggestion
    "@aave/math-utils": "1.24.0",
```

**Review by JoaquinBattilana — DISMISSED** [review]

**Review by foodaka — APPROVED** [review]

**Review by foodaka — APPROVED** [review]

**Review by JoaquinBattilana — APPROVED** [review]

**Pull request merged**